test/Lachain.CoreTest/RPC/HTTP/Web3/AccountServiceWeb3Test.cs

```diff
@@ -136,30 +150,74 @@ public void Test_GetTransactionCount()
             var ethTx = new TransactionChainId(rawTx2.HexToBytes());
             var address = ethTx.Key.GetPublicAddress().HexToBytes().ToUInt160();
 
-            var txCountBefore = _apiService!.GetTransactionCount(ethTx.Key.GetPublicAddress(), "latest");        
+            var txCountBefore = _apiService!.GetTransactionCount(ethTx.Key.GetPublicAddress(), "latest");
             Assert.AreEqual(txCountBefore, 0);
 
-            Execute_dummy_transaction();
+            Execute_dummy_transaction(true);
 
             var txCountAfter = _apiService!.GetTransactionCount(ethTx.Key.GetPublicAddress(), "latest");
             Assert.AreEqual(txCountAfter, 1);
         }
 
+        [Test]
+        // Changed GetTransactionCount to public
+        public void Test_GetTransactionCount_pending()
+        {
+
+            var rawTx2 = "0xf8848001832e1a3094010000000000000000000000000000000000000080a4c76d99bd000000000000000000000000000000000000000000042300c0d3ae6a03a0000075a0f5e9683653d203dc22397b6c9e1e39adf8f6f5ad68c593ba0bb6c35c9cd4dbb8a0247a8b0618930c5c4abe178cbafb69c6d3ed62cfa6fa33f5c8c8147d096b0aa0";
+            var ethTx = new TransactionChainId(rawTx2.HexToBytes());
+            var address = ethTx.Key.GetPublicAddress().HexToBytes().ToUInt160();
+
+            var txCountBefore = _apiService!.GetTransactionCount(ethTx.Key.GetPublicAddress(), "pending");
+            Assert.AreEqual(txCountBefore, 0);
+
+            TransactionUtils.SetChainId(41);
```

Comment:
can we get the chainId from the config.json and use it; like we did in the `getCode` ?